modules/abstract-cosmos/src/cosmosCoin.ts

```diff
@@ -578,7 +578,9 @@ export class CosmosCoin extends BaseCoin {
     if (response.status !== 200) {
       throw new Error('Account not found');
     }
-    if (response.body.account['@type'] == ('/ethermint.types.v1.EthAccount' || '/injective.types.v1beta1.EthAccount')) {
+    if (
+      ['/ethermint.types.v1.EthAccount', '/injective.types.v1beta1.EthAccount'].includes(response.body.account['@type'])
```

Comment:
Two things here:
- define these string constants somewhere
- add a test for this change